bin/nvm_install

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash -eu
 
+# shellcheck source=/dev/null
+source "$HOME"/.nvm/nvm.sh
+
 # This script expects a valid .nvmrc at the project's root
 if [ ! -f .nvmrc ]; then
     echo "No valid .nvmrc file found"
     exit 1
 fi
 
-nvm install
-nvm use
+nvm install
```

Comment:
Please add a newline at end of file to avoid issues with `cat` and other POSIX commands (also most IDEs will add one automatically next time we open the file in an editor, and will thus mark the file as modified next time it gets opened because of this CRLF at EOF being added by them automatically for POSIX conventions)